autocomplete/definitions/namedTypes/tes3vector2/distance.lua

```diff
@@ -0,0 +1,8 @@
+return {
+	type = "method",
+	description = [[Calculates the distance to another vector in the standard way.]],
```

Comment:
You can specify it's Euclidean distance. In addition, you can link to the related Wikipedia article. The following will be rendered as a link with text in [ ].
```md
[Euclidean distance](https://en.wikipedia.org/wiki/Euclidean_distance)
```